suite-native/module-accounts-import/src/components/SelectableNetworkList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { A, pipe } from '@mobily/ts-belt';
+
+import { Box, Card, Text, VStack } from '@suite-native/atoms';
+import {
+    Network,
+    networks,
+    NetworkSymbol,
+    getMainnets,
+    getTestnets,
+} from '@suite-common/wallet-config';
+import { mainnetsOrder, testnetsOrder } from '@suite-native/config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { SelectableNetworkItem } from './SelectableNetworkItem';
+
+type SelectableAssetListProps = {
+    onSelectItem: (networkSymbol: NetworkSymbol) => void;
+};
+
+const listItemTitleStyle = prepareNativeStyle(utils => ({
+    marginBottom: utils.spacings.medium,
+}));
+
+const mainnets: Network[] = getMainnets();
+const testnets: Network[] = getTestnets();
+
+const mapAndSortNetworkItems = (networkItems: Network[], networkOrder: NetworkSymbol[]) =>
+    pipe(
+        networkItems,
+        A.map(network => ({
+            cryptoCurrencySymbol: network.symbol,
+            cryptoCurrencyName: networks[network.symbol].name,
+            iconName: network.symbol,
+            order: networkOrder.includes(network.symbol)
+                ? networkOrder.indexOf(network.symbol)
+                : Number.MAX_SAFE_INTEGER,
+        })),
+        A.sort((a, b) => (a.order > b.order ? 1 : -1)),
+    );
+
+export const SelectableNetworkList = ({ onSelectItem }: SelectableAssetListProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const getSelectableNetworkItems = (networkItems: Network[], networkOrder: NetworkSymbol[]) => {
+        const sortedNetworkItems = mapAndSortNetworkItems(networkItems, networkOrder);
```

Comment:
This will run whole generation process with every rerender. It's not big deal but I would go with just simple  two variables before component like:

```
const sortedMainnetsNetworks = mapAndSortNetworkItems(mainnets, mainnetsOrder);
const sortedTestnetNetworks = mapAndSortNetworkItems(testnets, testnetsOrder);
```

and then use them directly, you will also be able to get rid this `getSelectableNetworkItems ` and map it directly in render.

Reply:
f70c4faa99a5dd7598596b6b7a30aefbe99ac910